mmv1/products/ces/Tool.yaml

```diff
@@ -103,6 +103,47 @@ properties:
         type: NestedObject
         description: Represents a select subset of an OpenAPI 3.0 schema object.
         properties:
+          - name: additionalProperties
+            type: String
+            description: |-
+              Optional. Defines the schema for additional properties allowed in an object.
+              The value must be a valid JSON string representing the Schema object.
+              (Note: OpenAPI also allows a boolean, this definition expects a Schema JSON).
+            state_func: 'func(v interface{}) string { s, _ := structure.NormalizeJsonString(v);
+              return s }'
+            custom_flatten: 'templates/terraform/custom_flatten/json_schema.tmpl'
+            custom_expand: 'templates/terraform/custom_expand/json_value.tmpl'
+            validation:
+              function: 'validation.StringIsJSON'
+            required: false
```

Comment:
nit: explicitly marking it `required: false` is not necessary since this is the default action when not set.